gulpfile.js

```diff
@@ -15,16 +17,38 @@ var lintPaths = [
     'gulpfile.js'
 ];
 
-gulp.task('test', function(done) {
+gulp.task('serve:backend', function () {
+    process.env.PYTHONUNBUFFERED = 1;
+    process.env.PYTHONDONTWRITEBITECODE = 1;
+    spawn('python', ['manage.py', 'runserver'], {
```

Comment:
By default runserver binds to localhost, so this doesn't work in a docker container. I got this working by adding a `0.0.0.0:8000` to the args list, but ideally it that could be a default overridden by an environment variable